### PR TITLE
Features/26 forget jutsu

### PR DESCRIPTION
**Pull request by NetzachSephira**

### CODE REVIEW

**Review by levimeahan — COMMENTED** [review]
Looks good, but let's add a confirm dialog so people don't accidentally forget their jutsu 👍

**Comment by NetzachSephira** [comment]
Added it in the same way as with the clan page and resigning from a position to remain consistent.

File: equip.php [hunk 1]
```diff
@@ -338,7 +339,53 @@ function jutsu() {
...
+				throw new Exception("Invalid Jutsu!");
+			}
+
+			//Checking if player has jutsu that depend on the jutsu he's trying to forget.
+			$can_forget = userHasChildrenJutsu($jutsu_id, $player);
+			if($can_forget == false){
+				throw new Exception("You cannot forget the parent of a jutsu you know!");
+			}
+
+            if($_POST['confirm_forget']) {
```
Comment (levimeahan) [inline equip.php:356]:
```suggestion
            if(!empty($_POST['confirm_forget'])) {

```

File: equip.php [hunk 2]
```diff
@@ -502,4 +551,12 @@ function jutsu() {
...
 
+function userHasChildrenJutsu($id, $player){
+    foreach($player->jutsu as $element){
+        if($id == $element->parent_jutsu){
+            return false;
+        }
+    }
 
+    return true;
+}
```
Comment (levimeahan) [inline equip.php:562]:
```suggestion
}

```

**Review by levimeahan — APPROVED** [review]
LGTM!

**Pull request merged**